src/main/windows.ts

```diff
@@ -49,7 +49,7 @@ export const createHappWindow = async (
       if (!relativeFilePath.endsWith('index.html') && !fallbackToIndexHtml) {
         return net.fetch(url.pathToFileURL(absoluteFilePath).toString());
       } else {
-        const indexHtmlResponse = await net.fetch(url.pathToFileURL(absoluteFilePath).toString());
```

Comment:
This would have still tried to fetch the wrong path here instead of the index.html in case that `absoluteFilePath` cannot be found on the filesystem.